keyboards/asmodus/plaquead/rgb/rgb.c

```diff
@@ -0,0 +1,118 @@
+//  { 224, 180},              { 196, 180}, { 182, 180}, { 168, 180}, { 154, 180}, { 140, 180}, { 126, 180}, { 112, 180}, {  98, 180}, {  84, 180}, {  70, 180}, {  56, 180},                           {  14, 180},
+//  {   0, 225}, {  14, 225}, {  28, 225},                                        {  84, 225},                           { 126, 225}, { 140, 225}, { 154, 225}, { 168, 225},              { 196,  45}, { 210,  45}, { 224,  90}
+}, {
+    4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          ,
+    4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 1          , 1          , 1          , 4          , 4          , 4          ,
+    4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          ,
+    1          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 1          ,
+    4          , 1          , 1          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 4          , 1          , 1          , 4          ,
+    1          , 1          , 1          ,                                        4          , 4                       , 1          , 1          , 1          , 1          , 1          , 1          , 1          , 1          ,
+
+
+    2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2           ,
+    2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2          , 2                 
+
+} };
+#endif
+
+#if defined(RGB_MATRIX_ENABLE) && defined(CAPS_LOCK_LED_INDEX) && defined(SCROLL_LOCK_LED_INDEX)
+
+#if !defined(CAPS_LOCK_MAX_BRIGHTNESS)
+    #define CAPS_LOCK_MAX_BRIGHTNESS RGB_MATRIX_MAXIMUM_BRIGHTNESS
+#endif
+
+#if !defined(CAPS_LOCK_VAL_STEP)
+    #define CAPS_LOCK_VAL_STEP RGB_MATRIX_VAL_STEP
+#endif
+
+#if !defined(SCROLL_LOCK_MAX_BRIGHTNESS)
+    #define SCROLL_LOCK_MAX_BRIGHTNESS RGB_MATRIX_MAXIMUM_BRIGHTNESS
+#endif
+
+#if !defined(SCROLL_LOCK_VAL_STEP)
+    #define SCROLL_LOCK_VAL_STEP RGB_MATRIX_VAL_STEP
+#endif
+
+
+bool rgb_matrix_indicators_user(void) {
```

Comment:
User functions should be used only in keymap.c/user space, and kb functions should be used in keyboard code (such as this):
```suggestion
bool rgb_matrix_indicators_kb(void) {
    if (!rgb_matrix_indicators_user()) {
        return false;
    }
```